images/java/Dockerfile-java-debian

```diff
@@ -0,0 +1,48 @@
+# ---- Build Stage ----
+FROM openjdk:17-jdk-slim AS build
+
+# Set up environment variables for Maven
+ENV MAVEN_VERSION 3.9.5
+ENV MAVEN_HOME /usr/share/maven
+
+# Update and install required packages
+RUN apt-get update && \
+    apt-get install -y curl && \
+    mkdir -p $MAVEN_HOME
+
+RUN curl -fsSL https://dlcdn.apache.org/maven/maven-3/$MAVEN_VERSION/binaries/apache-maven-$MAVEN_VERSION-bin.tar.gz | tar -xzC $MAVEN_HOME --strip-components=1
```

Comment:
Could copy in maven wrapper like you did above

Reply:
Instead of installing like this

Or at least, I think it makes sense to be consistent between the two examples in how we're copying in maven.